netlify/functions/favorites.ts

```
@@ -0,0 +1,32 @@
+import { Handler } from '@netlify/functions';
+import { storage } from '../../server/storage';
+import { insertFavoriteSchema } from '../../shared/schema';
+
+export const handler: Handler = async (event) => {
+  try {
+    if (event.httpMethod === 'GET') {
+      const userId = event.queryStringParameters && event.queryStringParameters.userId;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)